### PR TITLE
Rewrite skinning tutorial main page and osu!catch

### PR DESCRIPTION
**Pull request by MegaApplePi**
Moves the `Skinning_Tutorial` folder out of the `Skinning` folder and deletes all of the assets used in the Skinning Tutorial (they will no longer be used). Begins rewrite for Skinning Tutorial. Rewrites the osu!catch skinning tutorial (completed).

Side note: The other modes are slowly making some progress (to be opened in another pull request).

---

### CODE REVIEW

File: wiki/Skinning_Tutorial/osu!catch/en.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+![](img/margin-16-10.jpg "The margins in 16:10 ratio aspect")
+
+![](img/margin-21-9.jpg "The margins in 21:9 ratio aspect")
+
+In 16:9 and 21:9, there should be enough room to place an image inside the margins (in red) on either side, but 16:10 might not.
+
+### Area of focus
+
+Don't make a big scorebar that covers most of the top of the playfield. Remember that the fruits fall down from the top of the screen and it could shorten the field of view drastically if the scorebar is too big. Most players are looking from the combo number and above to position the catcher, if their view is blocked, it may deter them from using your skin.
```
[the remark this answers is not in the document]
Comment (MegaApplePi) [reply wiki/Skinning_Tutorial/osu!catch/en.md]:
```diff
- Don't make a big scorebar
+ Avoid making a big scorebar
```
good enough?

[the remark this answers is not in the document]
Comment (RockRoller01) [reply wiki/Skinning_Tutorial/osu!catch/en.md]: yes

File: wiki/Skinning_Tutorial/en.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Skinning Tutorial
+
+*See also: [Skinning](/wiki/Skinning).*
+
+*This article is the tutorial for skinning. For frequently answered skinning questions, see [Skinning/FAQ](/wiki/Skinning/FAQ).*
+
+## Getting Started
+
+To get started, open the game client, select [Options](/wiki/Options), and click `Open osu! folder`. Once the file explorer opens, click on `Skins/` and create a new folder. You can name it whatever you want, but keep in mind that osu! will use this name when it is displayed in the options. Inside this folder, you can add any skinning elements listed in the skin element lists.
```
Comment (Joehuu) [inline wiki/Skinning_Tutorial/en.md]: `file explorer` -> `directory`?

`osu! will use this name when it is displayed in the options` -> `osu! will display this name in the options`

File: wiki/Skinning_Tutorial/en.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    -   [Photoshop](https://www.adobe.com/products/photoshop.html) (Paid)
+-   Audio editors
+    - [Audacity](https://www.audacityteam.org) (Free)
+    - [FL Studio](https://www.image-line.com/flstudio/) (Free/Paid)
+
+---
+
+The next thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
+
+The last thing to have ready is patience and a lot of time. Yes, this is very important as new skins (unfortunately) do not complete in a snap. Making a new skin can take a week and up to months or more, so give it some time.
```
Comment (Joehuu) [inline wiki/Skinning_Tutorial/en.md]: Remove `Yes`.

`and up to months or more` -> `and up to a month or more`

File: wiki/Skinning_Tutorial/en.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## What's next?
+
+From here, you can either checkout the [skinning elements lists](/wiki/skinning#skin-elements-lists) to get an idea for what to include in your skin or go through the some of the tutorials:
```
Comment (Joehuu) [inline wiki/Skinning_Tutorial/en.md]: `checkout` -> `check out`?

**Pull request merged**